deco.ts

```diff
@@ -28,6 +27,10 @@ const config = {
       dir: "admin",
       name: "deco-sites/admin",
     },
+    {
+      dir: "files",
+      name: "deco-sites/files",
+    },
```

Comment:
why?

Reply:
I just copied it as it was done in the admin, but I reversed it